shop3/explicit-stack-search/plan-tree.lisp

```diff
@@ -92,7 +92,7 @@ Particularly useful for structures, but could be generally applicable."
   (print-unreadably (d str)
     (format str "Primitive: ~S"
             (or (tree-node-expanded-task d) (tree-node-task d)))
-    #+ignore(when (tree-node-dependencies d)
+    (when (tree-node-dependencies d)
```

Comment:
I had hacked this out because it made the object unbearably verbose.  Should we control this printing somehow, or disable this and leave it for `describe`?